packages/web/src/hooks/useManagedAccountNotAllowedRedirect.ts

```diff
@@ -0,0 +1,51 @@
+import { useContext, useEffect } from 'react'
+
+import { useIsManagedAccount } from '@audius/common/hooks'
+
+import { ToastContext } from 'components/toast/ToastContext'
+import { FEED_PAGE } from 'utils/route'
+
+import { useNavigateToPage } from './useNavigateToPage'
+
+const messages = {
+  unauthorized: `You can't do that as a managed user`
```

Comment:
nit: period? :grin:

Reply:
let's see in QA. i think we've only done period when it's a multi sentence toast